spec/features/schools/participants/add_participants/add_ect_as_mentor_spec.rb

```diff
@@ -19,45 +19,47 @@
   end
 
   scenario "Induction tutor tries to add an ECT that's a mentor in another school" do
-    when_i_navigate_to_participants_dashboard
-    when_i_click_to_add_a_new_ect_or_mentor
-    then_i_should_be_on_the_who_to_add_page
+    outside_auto_assignment_window do
+      when_i_navigate_to_participants_dashboard
+      when_i_click_to_add_a_new_ect_or_mentor
+      then_i_should_be_on_the_who_to_add_page
 
-    when_i_select_to_add_a "Mentor"
-    when_i_click_on_continue
-    then_i_am_taken_to_the_what_we_need_to_know_about_this_mentor_page
+      when_i_select_to_add_a "Mentor"
+      when_i_click_on_continue
+      then_i_am_taken_to_the_what_we_need_to_know_about_this_mentor_page
 
-    when_i_click_on_continue
-    then_i_am_taken_to_add_mentor_full_name_page
+      when_i_click_on_continue
+      then_i_am_taken_to_add_mentor_full_name_page
 
-    when_i_add_mentor_name
-    when_i_click_on_continue
-    then_i_am_taken_to_add_teachers_trn_page
+      when_i_add_mentor_name
+      when_i_click_on_continue
+      then_i_am_taken_to_add_teachers_trn_page
 
-    when_i_add_the_trn
-    when_i_click_on_continue
-    then_i_am_taken_to_add_date_of_birth_page
+      when_i_add_the_trn
+      when_i_click_on_continue
+      then_i_am_taken_to_add_date_of_birth_page
 
-    when_i_add_a_date_of_birth
-    when_i_click_on_continue
-    then_i_am_taken_to_add_ect_or_mentor_email_page
+      when_i_add_a_date_of_birth
+      when_i_click_on_continue
+      then_i_am_taken_to_add_ect_or_mentor_email_page
 
-    when_i_add_ect_or_mentor_email
-    when_i_click_on_continue
-    then_i_am_taken_to_mentor_start_training_page
+      when_i_add_ect_or_mentor_email
+      when_i_click_on_continue
+      then_i_am_taken_to_mentor_start_training_page
 
-    when_i_choose_summer_term_2023
-    when_i_click_on_continue
-    then_i_am_taken_to_choose_mentor_partnership_page
+      when_i_choose_summer_term_next_cohort
```

Comment:
I think the meaning in this spec should be "when_i_choose_summer_term_this_cohort" as this was only something done during the registration for 2023 when we didn't know for sure which cohort they would start in